### PR TITLE
Update 02-install-packages.sh

### PR DESCRIPTION
**Pull request by ScardracS-zz**
Removed grub2 as we don't need it and add compression of $OUT_NAME (fedora.img)

### CODE REVIEW

**Review by ScardracS — COMMENTED** [review]
Removed lightdm in favor of phosh

**Comment by Torbuntu** [comment]
> Removed lightdm in favor of phosh

This was discussed before, and is not preferred, because currently Phosh does not have a mechanism for switching to other environments (such as GNOME). If we happen to package and implement Plasma Mobile in the future, this will be a useful feature. We do not want to remove lightdm for that reason as a default.

**Comment by ScardracS** [comment]
I thought gnome was only a "side effect" and not an intended thing. Ok so, thanks for the clarification

File: phone-scripts/02-install-packages.sh [hunk 1]
```diff
@@ -16,32 +16,34 @@ errecho () {
...
+infecho "Removing unneeded packages..."
+dnf remove grub2 -y
+rm -rf rootfs/boot/grub2
+rm -rf rootfs/etc/grub.d
+rm rootfs/etc/grub2-efi.cfg
+
+infecho "Enabling phosh boot..."
+systemctl disable gdm.service
 systemctl disable initial-setup.service
-systemctl enable lightdm
```
Comment (Torbuntu) [inline phone-scripts/02-install-packages.sh:40]: I think we want to keep lightdm so as to be able to switch to other environments such as GNOME.

**Pull request closed without merging**